src/pages/thank-you.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from 'gatsby';
+
+const ThankYou = () => {
+  return (
+    <div style={{
+      display: 'flex',
+      flexDirection: 'column',
+      width: '100vw',
+      height: '100vh',
+      alignItems: 'center',
+      justifyContent: 'center'
+    }}>
```

Comment:
I would suggest rewriting this with reusable components from 'bricks'.
If you need style customization on bricks, you may use styled-components and styled system.

@sachinmjadhav